usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/MedicationData.java

```diff
@@ -0,0 +1,105 @@
+package org.pesmypetcare.usermanagerlib.datacontainers;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import org.json.JSONObject;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class MedicationData {
+    private double quantity;
+    private int duration;
+    private int periodicity;
+
+    public MedicationData(double inQuantity, int inDuration, int inPeriodicity) {
+        this.quantity = inQuantity;
+        this.duration = inDuration;
+        this.periodicity = inPeriodicity;
+    }
+
+    /**
+     * Returns the quantity.
+     * @return Quantity of the medication
+     */
+    public double getQuantity() {
+        return quantity;
+    }
+
+    /**
+     * Sets a new quantity
+     * @param inQuantity New meal name
+     */
+    public void setMedicationQuantity(double inQuantity) {
+        this.quantity = inQuantity;
+    }
+
+    /**
+     * Returns the number of days of this medication.
+     * @return Amount of days.
+     */
+    public int getDuration() {
+        return this.duration;
+    }
+
+    /**
+     * Sets a new amount of days of this medication.
+     * @param inDuration Amount of kcalories.
+     */
+    public void setDuration(int inDuration) {
+        this.duration = inDuration;
+    }
+
+
+    /**
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First sentence should end with a period.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5403182)